### PR TITLE
Mention the simpler /types setup first

### PR DESCRIPTION
**Pull request by KnorpelSenf**
We already do something similar in the introduction.

### CODE REVIEW

File: site/docs/guide/api.md [hunk 1]
```diff
@@ -79,16 +79,25 @@ grammY ships with complete type coverage of the Bot API.
 The [`@grammyjs/types`](https://github.com/grammyjs/types) repository contains the type definitions that grammY uses internally.
 These type definitions are also exported so you can use them in your own code.
 
+#### Type Definitions on Deno
+
+On Deno, you can simply import type definitions from `types.ts` which is right next to `mod.ts`:
```
Comment (roj1512) [inline site/docs/guide/api.md:84]:
```suggestion
On Deno, you can simply import type definitions from `types.ts`, which is right next to `mod.ts`:
```

**Review by roj1512 — APPROVED** [review]
LGTM.

**Review by CikiMomogi — APPROVED** [review]
Poor Node.js :smiling_face_with_tear:

**Review by habemuscode — APPROVED** [review]

**Comment by roj1512** [comment]
Thank you, @habemuscode!

File: site/docs/es/guide/api.md [hunk 2]
```diff
@@ -79,16 +79,25 @@ grammY viene con una cobertura completa de tipos de la API del Bot.
...
+
+```ts
+import { type Chat } from "https://deno.land/x/grammy/types.ts";
+```
+
 #### Definiciones de tipos en Node.js
 
-En Node.js, necesitas importar los tipos desde `grammy/types`.
+On Node.js, things are more complicated.
+You need to import the types from `grammy/types`.
```
Comment (CikiMomogi) [inline site/docs/es/guide/api.md:93]: Oopss @habemuscode @roj1512

**Review by CikiMomogi — COMMENTED** [review]
You missed this one, capt.

**Pull request merged**